lib/features/prediction/domain/repositories/repo.dart

```diff
@@ -0,0 +1,10 @@
+
+
+import 'package:either_dart/either.dart';
+
+import '../../../../core/shared/shared.dart';
+import '../../prediction.dart';
+
+abstract class PredictionRepository {
+  Future<Either<Failure, PredictionEntity>> get prediction;
```

Comment:
Use this snippet as the correct method definition:

```dart
Future<Either<Failure, PredictionEntity>> fetch({
    required String fixtureGuid,
});
```

Reply:
Done